content/modules/tag-input/versions/1.5.2.um

```diff
@@ -0,0 +1,21 @@
+@version 1.5.2
+  @api Tag Input
+    @prototype hx.TagInput
+      @constructor
+        @param selector [String/HTMLElement]
+        @param? options [Object]
+          @property autocompleteData [Array/Function]
+            @added: @issue 194
+            @description:
+              @p: The data to use as suggestions for the tag input. Data can be specified as an array of items or as a function that returns an array of items.
+              @p: See @code[hx.AutoComplete] for further details.
+              @p: By default, no autocompletion is provided.
+          @property autocompleteOptions [Object]
+            @added: @issue 194
+            @description: Special options to be passed to the autocomplete. See @code[hx.AutoComplete] options.
```

Comment:
We generally add links for the module docs in this case. See the `autocompletePickerOptions` in `form-builder/versions/1.4.0.um` as an example